libs/designsystem/src/lib/components/chart/chart-js/chart-js.service.ts

```diff
@@ -280,9 +280,13 @@ export class ChartJSService {
     const typeConfig = this.chartConfigService.getTypeConfig(type);
 
     const labelsToApply = (() => {
-      if (labels?.length > 0) return labels;
-      else if (type === 'stock') return this.getDefaultStockLabels(datasets, this.locale);
-      else return this.createBlankLabels(datasets); // ChartJS requires labels
+      if (type === 'stock' && !Array.isArray(labels)) {
+        return this.getDefaultStockLabels(datasets, this.locale);
+      } else if (labels?.length > 0) {
+        return labels;
+      } else {
+        return this.createBlankLabels(datasets);
+      }
```

Comment:
I've taken the liberty of changing the logic from commit e2dccbfb26727ec3d9c91ea963b9e23fd9f5ff50. 

If an empty array was provided to a chart _not_ using the time scale it would result in a blank chart. Therefore i've switched it around to be "if the type is stock & no labels are provided - get default stock labels". Such that you can override the default stock labels by simply passing something into the `labels` input property.